src/Window.cpp

```diff
@@ -503,6 +488,7 @@ void Window::onMouseAlgoComboBoxChanged(QString name) {
     m_runStatus->setStyleSheet("");
     m_runOutput->clear();
     SettingsMisc::setRecentMouseAlgo(name);
+    stats->resetAll();
```

Comment:
Nitpick: do you mind moving this up a line, so that it's clear that we always reset the stats whenever we clear the run output?